src/promptflow/tests/sdk_cli_azure_test/_fake_credentials.py

```diff
@@ -0,0 +1,20 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from azure.core.credentials import AccessToken
+
+
+class FakeTokenCredential:
```

Comment:
Weird to put these files under `sdk_cli_azure_test`, can we put them in `tests/test_utilities` instead?

Reply:
that should only be needed inside pfazure test, so in new implementation, still put it under `sdk_cli_azure_test`